index.ts

```diff
@@ -1094,6 +1110,33 @@ commander
     });
   });
 
+/**
+ * Clasp run <function>
+ * 
```

Comment:
- Describe what this can and can't do. i.e. does it accept params?
- Remove newlines.